plugin/sqlsh/src/main/java/com/twosigma/beaker/sqlsh/utils/QueryExecutor.java

```diff
@@ -153,7 +153,7 @@ protected static String setObject(String sql, Object value){
       String begin = sql.substring(0, index);
       String end = sql.substring(index + 1, sql.length());
       if(value instanceof String){
-        ret = begin + "\'" + value + "\'" + end;
+        ret = begin + "'" + replaseString(((String) value), "'", "''", 0) + "'" + end;
```

Comment:
How can I change encoding of database to test? Does we support all
encodings?

3 Авг 2016 г. 7:30 PM пользователь "Scott Draves" <notifications@github.com>
написал:

> In
> plugin/sqlsh/src/main/java/com/twosigma/beaker/sqlsh/utils/QueryExecutor.java
> <https://github.com/twosigma/beaker-notebook/pull/4591#discussion_r73348115>
> :
>
> > @@ -153,7 +153,7 @@ protected static String setObject(String sql, Object value){
> >        String begin = sql.substring(0, index);
> >        String end = sql.substring(index + 1, sql.length());
> >        if(value instanceof String){
> > -        ret = begin + "\'" + value + "\'" + end;
> > +        ret = begin + "'" + replaseString(((String) value), "'", "''", 0) + "'" + end;
>
> that's a good question, what encoding are the queries?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/twosigma/beaker-notebook/pull/4591/files/bb1edbf8c66b097c661e46bc6be42e28b744ffd6#r73348115>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKH3aazcQ06oXWUUqtoodJ2NA3HVUD4Oks5qcLAjgaJpZM4JbcC8>
> .
>